### DISCUSSION TITLE
[CRITICAL] Dealing with Scams

### DISCUSSION THREAD

**Discussion by deivpaukst** [rustdesk/rustdesk#3669] [General]
### Description

I am aware that there is another issue open about adding a popup message to warn the user that they might be scammed (which honestly should have been added in some form by now as it has been months and more people could have fallen to scams by the use of your software). The nice thing about being an open source project is that suggestions are a lot more likely to be looked at and considered. I think that adding a header at the top of the website in **BIG BOLD LETTERS** where the first word is scam, would be a good start, because the people who are looking for a remote access program are already most likely aware of what can be done with it and would take the warning as a team of developers putting in effort to help not only people who may use their software but also random people who don't deserve to get their money stolen. 

### Impact

Users who might fall to internet scams.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/1675#issuecomment-1482934990
